packages/cli/src/models/status/StatusChecker.js

```diff
@@ -63,34 +69,33 @@ export default class StatusChecker {
   }
 
   async checkVersion() {
-    const observed = (await this.app()).version
+    const observed = this._project.version
     const expected = this.networkFile.version
```

Comment:
You are right, L72 won't work for lib projects, but as fas as I understand, we are not validating the version (and we were not validating it in the old model :thinking:) for lib projects. That being said, we should totally check it! But I would prefer to do it in a future PR as it would be a "new feature".